src/common/components/SkeletonScreen.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Center, Spinner } from '@chakra-ui/react';
+
+export const SkeletonScreen: React.FC = () => {
```

Comment:
You can use Skeleton component directly from chakra ui, u can check out it's implementation in ExecutionView

Reply:
Modified this component to include skeleton component from chakra UI and reused it in PreviewView and ExecuteView